notebooks/zx_intro.jl

```diff
@@ -11,8 +11,8 @@ begin
     Pkg.add("OpenQASM")
     Pkg.add("Vega")
     Pkg.add("DataFrames")
-    Pkg.add("YaoHIR.jl")
-    Pkg.add("ZXCalculus.jl")
+    Pkg.add("YaoHIR")
+    Pkg.add("ZXCalculus")
     Pkg.add(
         url = "https://gitlab.informatik.uni-bremen.de/qce/quantumequivalencechecker.jl",
         rev = "feat/zx",
```

Comment:
can we use Pluto builtin env instead of doing this?

Reply:
Sure